### ISSUE TITLE
[bug] button bug at tsprojecttodo/src/components /TodoItem.tsx

### ISSUE THREAD

**Issue by gonsura** [amitjimiwal/tsprojecttodo#2]
i think maybe dont put your onclick event at icon or svg tag, better  put on your ButtonWrapper component.
kamu dapat melakukan itu hanya dengan add one props

**Reason?**

1. _bad accessibility_,  one example is if a user wants to access your website because he feels it is a cool website but he doesn't have a mouse or trackpad, of course he will use tab to target the button, but if he targets the delete or edit button and presses enter the event will not happen
2. and the main reason is that the delete or edit button has padding, if the user does not click on the icon but around the padding the event will not occur

<img width="959" alt="Screenshot 2023-09-21 232322" src="https://github.com/amitjimiwal/tsprojecttodo/assets/121378129/ecad0d71-374b-44bb-93f2-d3e42eb7087b">

**Comment by gonsura** [comment]
yes please
